js/languages/en-US.json

```diff
@@ -806,6 +806,7 @@
   },
```

Comment:
I don't think it's properly removing coupons internally.

- add a coupon for 50% off
- add a coupon for 51% off

Correctly, gives you an error that coupons cannot bring a price below zero.

- remove the 50% off coupon
- attempt to add the 51% off coupon

Expectation: It should work now.
Observed: Gives you an error that coupons cannot take the price below zero.